app/src/main/res/layout/view_song_details.xml

```diff
@@ -0,0 +1,154 @@
+
+            <Space
+                style="@style/DetailsSpace"
+                android:layout_width="match_parent" />
+
+            <TextView
+                style="@style/DetailsLabel"
+                android:text="@string/details_album_artist" />
+
+            <TextView
+                android:id="@+id/album_artist"
+                style="@style/DetailsValue"
+                tools:text="TOOL" />
+
+            <Space
+                style="@style/DetailsSpace"
+                android:layout_width="match_parent" />
+
+            <TextView
+                style="@style/DetailsLabel"
+                android:text="@string/details_artist" />
+
+            <TextView
+                android:id="@+id/artist"
+                style="@style/DetailsValue"
+                tools:text="TOOL" />
+
+            <Space
+                style="@style/DetailsSpace"
+                android:layout_width="match_parent" />
+
+            <TextView
+                style="@style/DetailsLabel"
+                android:text="@string/details_album" />
+
+            <TextView
+                android:id="@+id/album"
+                style="@style/DetailsValue"
+                tools:text="Fear Incolulum" />
+
+            <Space
+                style="@style/DetailsSpace"
+                android:layout_width="match_parent" />
+
+            <TextView
+                style="@style/DetailsLabel"
+                android:text="@string/details_title" />
+
+            <TextView
+                android:id="@+id/title"
+                style="@style/DetailsValue"
+                tools:text="7emptest" />
+
+            <Space
+                style="@style/DetailsSpace"
+                android:layout_width="match_parent" />
+
+            <TextView
+                style="@style/DetailsLabel"
+                android:text="@string/details_disc_number" />
+
+            <TextView
+                android:id="@+id/disc_number"
+                style="@style/DetailsValue"
+                tools:text="1" />
+
+            <Space
+                style="@style/DetailsSpace"
+                android:layout_width="match_parent" />
+
+            <TextView
+                style="@style/DetailsLabel"
+                android:text="@string/details_track_number" />
+
+            <TextView
+                android:id="@+id/track_number"
+                style="@style/DetailsValue"
+                tools:text="9" />
+
+            <Space
+                style="@style/DetailsSpace"
+                android:layout_width="match_parent" />
+
+            <TextView
+                style="@style/DetailsLabel"
+                android:text="@string/details_duration" />
+
+            <TextView
+                android:id="@+id/duration"
+                style="@style/DetailsValue"
+                tools:text="15:45" />
+
+            <Space
+                style="@style/DetailsSpace"
+                android:layout_width="match_parent" />
+
+            <TextView
```

Comment:
I would suggest re-ordering these as:

- Title
- Album
- Year
- Artist
- Album Artist 
- Track Number
- Total Tracks
- Duration
- Path

Displaying the path also seems a bit ugly, but I'm OK to keep it if you have a use case.

Reply:
Done